tso-server/src/test/java/org/apache/omid/tso/TestTSOChannelHandlerNetty.java

```diff
@@ -72,12 +73,13 @@
 
     // Component under test
     private TSOChannelHandler channelHandler;
+    private final int port = NetworkUtils.availablePort();
 
     @BeforeMethod
     public void beforeTestMethod() {
         MockitoAnnotations.initMocks(this);
         TSOServerConfig config = new TSOServerConfig();
-        config.setPort(1434);
+        config.setPort(port);
```

Comment:
This doesn't mesh with the ticket description.
We are still using the same port for all tests. 
This may help when the 1434 port is unavailable for some other reason, but AFAICT the TSOChannelHandler lifecycle is the same as it was pre-patch.
If the problem is really what you describe, then we need to get a new port for every method.